### PR TITLE
Feature/ruby codegen

### PR DESCRIPTION
**Pull request by 115100**

### CODE REVIEW

**Comment by 115100** [comment]
/cc @dim 

Uhh. Accidentally included a python commit.

File: grpclb.gemspec [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Gem::Specification.new do |s|
+	s.name          = 'grpclb'
+	s.summary       = 'grpclb protocol'
+	s.version       = '0.1.0'
+	s.authors       = ['Black Square Media']
+	s.platform      = Gem::Platform::RUBY
+	s.files         = `git ls-files ruby`.split("\n")
+	s.require_paths = ['ruby/lib']
+
+	s.add_runtime_dependency 'google-protobuf'
```
Comment (dim) [inline grpclb.gemspec]: add `grpc-tools` as a dev dependency

File: Makefile [hunk 2]
```diff
@@ -42,6 +43,12 @@ proto.python: $(PYPROTO)
 
 	python -m grpc_tools.protoc -I. --python_out=python --grpc_python_out=python $<
 
+proto.ruby: $(RBPROTO)
+%_pb.rb: %.proto
+	@if ! which grpc_ruby_plugin > /dev/null ; then echo 'Run "gem install grpc-tools" required to generate ruby code'; exit 1; fi
```
Comment (dim) [inline Makefile]: no need if you add it to gemspec

File: grpclb.gemspec (hunk 1 above)
Comment (dim) [inline grpclb.gemspec]: You also need a Gemfile

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
+
```
Comment (dim) [inline Gemfile]: all you need is `gemspec`, that will use the deps from the gemspec

**Pull request merged**